projects/igniteui-angular-i18n/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "compilerOptions": {
+        "paths": {
+            "igniteui-angular": [
```

Comment:
Interestingly, this is defined in the root tsconfig. Would it work even without this one? Not a big deal, just interesting

Reply:
Oddly enough, not it finds the igniteui-angular mapping location correctly, even without the _projects/igniteui-angular-i18n/tsconfig.json_ file setting it relatively to the **i18n**.